src/features/Calendars/CalendarSlice.ts

```
@@ -193,9 +193,31 @@ const CalendarSlice = createSlice({
+              (eventId: string) => {
+                const eventIdBase = extractEventBaseUuid(eventId);
+                return eventIdBase === action.payload.event.uid;
+              }
+            )
+          ) {
+            Object.keys(state.list[action.payload.calId].events)
+              .filter((eventKey) => {
+                const baseUid = extractEventBaseUuid(eventKey);
+                return baseUid === action.payload.event.uid;
+              })
+              .forEach(
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)